src/main/java/org/jboss/logmanager/FileMove.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.jboss.logmanager;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ *
+ * @author panos
+ */
+public class FileMove {
```

Comment:
This should be moved to the `org.jboss.logmanager.handlers` package and be package-private.